editor/editor.tsx

```diff
@@ -601,9 +604,8 @@ function Editor(props: any) {
                             onClick={() => {
                                 if (code.length <= LIMIT_CLIPBOARD_LEN) {
                                     // copy the unique url to clipboard using `<input/>`
-                                    const url = `https://gosling-lang.github.io/gosling.js/?full=${isHideCode}&spec=${JSONCrush(
-                                        code
-                                    )}`;
+                                    const crushedSpec = encodeURIComponent(JSONCrush.crush(code));
```

Comment:
Just changed this part so that "SAVE URL" button on the Editor works properly.

(Reference of its breaking update: https://github.com/KilledByAPixel/JSONCrush/commit/0fc0305d507566bc7abd2dc9ce51c98f4af86f60)

Didn't know JSONCrush is now published at NPM which is great!